wan/RestApi/src/main/java/com/example/restapi/repoistory/MemoryFoodRepository.java

```diff
@@ -0,0 +1,40 @@
+package com.example.restapi.repoistory;
+
+import com.example.restapi.domain.Food;
+import org.springframework.stereotype.Repository;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Repository
+public class MemoryFoodRepository implements FoodRepository {
+    private static Map<Long, Food> store = new HashMap<>();
+
+    @Override
+    public void save(Food food) { //입력 받은 id로 음식 저장
+        store.put(food.getId(), food);
+    }
+
+    @Override
+    public Food findById(Long id) { //id(key)로 음식 찾기
+        return store.get(id);
+    }
+
+    @Override
+    public List<Food> findAll() { // 전체 리스트 찾기 (count 내림차순)
+        return store.values().
+                            stream().sorted(Comparator.comparing(Food::getCount).reversed()).toList();
+    }
```

Comment:
stream이 길어져서 줄바꿈이 필요할 때는 심볼 앞에서 내리는 컨밴션을 지킵니다!
```suggestion
    @Override
    public List<Food> findAll() { // 전체 리스트 찾기 (count 내림차순)
        return store.values().stream()
        .sorted(Comparator.comparing(Food::getCount)
        .reversed())
        .toList();
    }
```

Reply:
넵!